concourse/scripts/smoke_tests_email.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+DEPLOY_ENV=$1
+SYSTEM_DNS_ZONE_NAME=$2
+ALERT_EMAIL_ADDRESS=$3
+
+TO="${ALERT_EMAIL_ADDRESS}"
+FROM="${ALERT_EMAIL_ADDRESS}"
```

Comment:
Indeed the address has to be verified. Because of that, I'd say it's fine that the sender is the same as addressee. And because there are no other issues, merging.